packages/checkout/widgets-lib/src/widgets/add-funds/functions/isSquidNativeToken.ts

```diff
@@ -0,0 +1,3 @@
+export const SQUID_NATIVE_TOKEN = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee';
```

Comment:
let's put this in `config.ts`?

Reply:
Remove export to only be used in this function